### PR TITLE
Sanitize postgresql page

### PR DESCRIPTION
**Pull request by andreipope**
@harsh-px when you have some time.

### CODE REVIEW

**Comment by harsh-px** [comment]
Let's finalize the template for the cassandra PR first. Until then pause on all other applications.

**Comment by vilasd** [comment]
@harsh-px can you please review this ??

File: content/portworx-install-with-kubernetes/application-install-with-kubernetes/postgresql/_index.md [hunk 1]
```diff
@@ -167,20 +289,24 @@ pgbench=# \d
...
+Next, list databases:
+
+```text
+\l
+```
 
-pgbench=# \q
+You should see something similar to this:
+
+![postgresql-psql](/img/postgresql-psql.png)
```
Comment (nathan-px) [inline content/portworx-install-with-kubernetes/application-install-with-kubernetes/postgresql/_index.md]: Drop the pictures, the text output was better.

File: content/portworx-install-with-kubernetes/application-install-with-kubernetes/postgresql/_index.md [hunk 2]
```diff
@@ -1,41 +1,61 @@
...
-Create the file`px-postgres-sc.yaml` with the following:
+The diagram below shows the main components of a highly available PostgreSQL with _Portworx_ deployment on top of Kubernetes:
 
-```text
+![postgresql-with-portworx-on-kubernetes](/img/postgresql-with-portworx-on-kubernetes.png)
+
+
+## Creating a StorageClass
 
+Let's begin by creating a file named `px-postgres-sc.yaml` with the following content:
```
Comment (nathan-px) [inline content/portworx-install-with-kubernetes/application-install-with-kubernetes/postgresql/_index.md]: Format these as a task.

**Comment by nathan-px** [comment]
@andreipope This was an improvement over the existing page, but there's a lot of conflicts left and it requires more work to bring it in line with the new style guide/shared file locations. I'm going to close this PR and create a new PD ticket to revise this leveraging whatever we've done here next time.

**Pull request closed without merging**